packages/core/upload/admin/src/hooks/tests/useAssets.test.js

```diff
@@ -0,0 +1,120 @@
+/* eslint-disable import/order */
```

Comment:
Because the `import` statements have to come after the jest mocks. At least this is how it was in the CJS era: did it change? I'd be more than happy to clean that up.

Reply:
Fixed in 1af6ad111e5a663909e54ff86912e5f84d6dc58b